main.go

```diff
@@ -86,49 +101,44 @@ func main() {
 	go handleInterrupt(exit)
 
 	// Load emotes before starting server.
-	var err error
 	chat, err = newChatRoom()
 	if err != nil {
 		common.LogErrorln(err)
 		os.Exit(1)
 	}
 
-	if addr == "" {
-		addr = settings.ListenAddress
+	if args.Addr == "" {
+		args.Addr = settings.ListenAddress
 	}
 
-	if rtmpAddr == "" {
-		rtmpAddr = settings.RtmpListenAddress
+	if args.RtmpAddr == "" {
+		args.RtmpAddr = settings.RtmpListenAddress
 	}
 
 	// A stream key was passed on the command line.  Use it, but don't save
 	// it over the stream key in the settings.json file.
-	if sKey != "" {
-		settings.SetTempKey(sKey)
+	if args.StreamKey != "" {
+		settings.SetTempKey(args.StreamKey)
 	}
 
 	common.LogInfoln("Stream key: ", settings.GetStreamKey())
 	common.LogInfoln("Admin password: ", settings.AdminPassword)
-	common.LogInfoln("HTTP server listening on: ", addr)
-	common.LogInfoln("RTMP server listening on: ", rtmpAddr)
+	common.LogInfoln("HTTP server listening on: ", args.Addr)
+	common.LogInfoln("RTMP server listening on: ", args.RtmpAddr)
 	common.LogInfoln("RoomAccess: ", settings.RoomAccess)
 	common.LogInfoln("RoomAccessPin: ", settings.RoomAccessPin)
 
 	rtmpServer := &rtmp.Server{
 		HandlePlay:    handlePlay,
 		HandlePublish: handlePublish,
-		Addr:          rtmpAddr,
+		Addr:          args.RtmpAddr,
 	}
 
 	router := http.NewServeMux()
 
 	router.HandleFunc("/ws", wsHandler) // Chat websocket
-	router.HandleFunc("/static/js/", wsStaticFiles)
-	router.HandleFunc("/static/css/", wsStaticFiles)
-	router.HandleFunc("/static/img/", wsImages)
-	router.HandleFunc("/static/main.wasm", wsWasmFile)
+	router.Handle("/static/", http.FileServer(http.FS(staticFsys)))
```

Comment:
This would only serve static files from the embedded filesystem and not look for anything on-disk, right?  I'm not sure I like that.  Changing a template or style would require recompiling the server to see changes.

I'm OK with embedding and using defaults, but I'd like it to look on-disk for non-default files first.

Reply:
staticFsys is a file.FS class which looks at filesystem first and then if it doesn't exist, it looks at the static dir.
https://github.com/zorchenhimer/MovieNight/blob/embed/main.go#L76
https://github.com/zorchenhimer/MovieNight/blob/embed/files/files.go#L33
I also double checked with moving the exe to a different directory and then doing `movienight.exe --write-static`, changing main.html, and then `movienight.exe` and it showed the changes
![image](https://user-images.githubusercontent.com/27922090/163680004-10d4e09f-7106-495e-9c10-0f86cf9f11c4.png)